src/main/java/org/mule/extension/http/internal/request/HttpPollingSource.java

```diff
@@ -114,7 +115,7 @@
 
   @Override
   protected void doStart() throws MuleException {
-    LOGGER.error("Starting source");
+    LOGGER.debug("Starting HTTP Polling Source");
```

Comment:
I think the purpose of Rodro's original comment was to add the name of the flow in which it is contained

Reply:
I think this is outdated, because I added after talking with Nico about how to get the Location

Reply:
With latest push now it's appearing as it was.. weird O.o